### DISCUSSION TITLE
TS80P glows red hot and the temperature fluctuates, then goes down.

### DISCUSSION THREAD

**Discussion by sigwinch28** [Ralim/IronOS#1022] [Q&A] [answered]
My TS80P has started malfunctioning out of the box after flashing v2.15.
The reported temperature climbs slowly, then stops somewhere in the 200-250C range and fluctuates.
Without cutting the power, the tip will get red hot, at which point I always pull the power.
The tip has a resistance of about 4.5 ohms and the reported tip temperature in the debug menu is approximately room temperature.

Is the tip broken, or is there something wrong with the iron? The PCB revision (printed on the board) is v1.2 and the "automatic" display orientation always shows "upside down".

I've split this from #1021 because this is more of an iron support question instead of an issue with IronOS at the moment.

**Comment by Ralim** [comment, marked as answer]
The most likely thing is an issue in the construction of the tip.
Normally the tips are created with a thermocouple built in by how the assemble one of the joints.
This is used to measure the offset from ambient temperature, to allow measuring the tip temp.

So if this thermocouple _broke_ or wasnt there, then you would have exactly this behaviour you are seeing.

If you try measuring the uV (or even mV) across the tip, and when its hot you should read higher than when its cold.
